blocks/form/form.js

```diff
@@ -462,7 +462,8 @@ export async function createForm(formDef, data) {
   if (formDef.appliedCssClassNames) {
     form.className = formDef.appliedCssClassNames;
   }
-  await generateFormRendition(formDef, form);
+  const formId = extractIdFromUrl(formPath); // formDef.id returns $form after getState()
```

Comment:
This shouldn't be required. We should have the form id in the html too.
Also, in future we should improve getState to return the actual form id. $form is a bug.

Reply:
Yeah seems like a bug in getState()

> We should have the form id in the html too.

But html has not been created yet right. `generateFormRendition` needs formId

Reply:
Yeah..got that. my comment was in relation to my last comment where I was assuming that html has the correct relationship as we are creating it.